packages/components/form/src/phone_field/phone_field.tsx

```diff
@@ -0,0 +1,91 @@
+import {
+  isPossiblePhoneNumber,
+  parsePhoneNumberFromString,
+} from "libphonenumber-js";
+import { useFormContext, Controller, RegisterOptions } from "react-hook-form";
+import snakeCase from "lodash/snakeCase";
+import { useTranslation } from "react-i18next";
+
+import { FormField } from "../form_field";
+import { useFormField } from "../use_form_field";
+import PhoneInput, { PhoneInputProps } from "./phone_input";
+
+interface OwnProps {
+  defaultValue?: string;
+  label?: Maybe<string>;
+  name: string;
+  rules?: Exclude<RegisterOptions, "valueAsNumber" | "valueAsDate">;
+}
+
+export type PhoneFieldProps = OwnProps &
+  Omit<PhoneInputProps, "onChange" | "defaultValue">;
+
+const formattedValue = (value: string): string => {
+  const parsedPhoneNumber = parsePhoneNumberFromString(value);
+  return parsedPhoneNumber?.formatNational() ?? "";
+};
+
+export const PhoneField: React.FC<PhoneFieldProps> = ({
+  defaultValue = "",
+  label: labelProp,
+  name,
+  rules,
+  width = "input",
+  zIndex = "formField",
+  ...props
+}): React.ReactElement => {
+  const { control } = useFormContext();
+  const { t } = useTranslation(["anchorForms", "forms"]);
+  const {
+    error,
```

Comment:
When @tute merges his work in https://github.com/BuoySoftware/anchor-ui/pull/139 we'll want to include the `hint` as well here.

Reply:
sounds good. I can hold off for that!